src/bicep/examples/appServicePlan/appService.bicep

```diff
@@ -0,0 +1,50 @@
+/*
+Deployes a web server farm(aka: App Service Plan) to support web container deployments for Linux.  
+Optionally enable dynamic auto scaling based on CPU Percentages using 'enableAutoScale' true/false
+*/
+targetScope = 'subscription'
+
+param mlzDeploymentVariables object = json(loadTextContent('../deploymentVariables.json'))
+
+@description('The name of the web server farm which will be created.  If unchanged or not specified, the MLZ resource prefix + "--ASP" will be utilized.')
+param appServicePlanName string = '${mlzDeploymentVariables.mlzResourcePrefix.Value}-asp'
+
+@description('The name of the resource group in which the app service plan will be deployed. If unchanged or not specified, the MLZ shared services resource group is used.')
+param targetResourceGroup string = '${mlzDeploymentVariables.spokes.Value[2].resourceGroupName}'
+
+@description('If true, enables dynamic scale-in & scale-out based on CPU percentages.  If false, then compute instances remain static with 2 instances supporting all traffic')
+param enableAutoScale bool = true
+
+var targetSubscriptionId_Var = targetResourceGroup == '${mlzDeploymentVariables.spokes.Value[2].resourceGroupName}' ? '${mlzDeploymentVariables.spokes.Value[2].subscriptionId}' : subscription().subscriptionId
+var location = deployment().location
+var kind = 'linux'
+var capacity = 2
+var sku = 'premium'
```

Comment:
The `sku` variable is mapped to the `tier` in the resource provider.  Customers may need to look up valid values for tier in their subscriptions if they get an error because a tier or machine SKU is not available. Maybe remove `tier` altogether as a parameter to the `Microsoft.Web/serverfarms@2020-12-01` provider because it is not required, and repurpose the `sku` parameter to map to the `name` parameter on the `Microsoft.Web/serverfarms@2020-12-01` provider? For example, allow someone to provide `B1` as the `sku` parameter, which would map to the `name` parameter on the provider.

Reply:
Nice call out, just submitted a commit that updates as requested.  Thank you